File: app/src/main/java/com/hda/bachelorandroid/Activity.java
package com.hda.bachelorandroid;

public class Activity {
    private String name;
    private String duration;
    private Boolean finished;
    private String _id ;

    public Activity(String name, String duration, String _id) {
        this.name = name;
        this.duration = duration;
        this.finished = false;
        this._id = _id;
    }
    public String get_id() { return _id;}

    public void set_id(String _id) {this._id = _id;
    }
    public String getDuration() {
        return duration;
    }

    public void setDuration(String duration) {
        this.duration = duration;
    }

    public Boolean getFinished() {
        return finished;
    }

    public void setFinished(Boolean finished) {
        this.finished = finished;
    }

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }
}

File: app/src/main/java/com/hda/bachelorandroid/services/activity/model/Activity.kt
package com.hda.bachelorandroid.services.activity.model

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

class Activity {

    @SerializedName("name")
    @Expose
    var name: String? = null
    @SerializedName("_id")
    @Expose
    var id: String? = null
    @SerializedName("activityId")
    @Expose
    var activityId: String? = null
    @SerializedName("duration")
    @Expose
    var duration: Int? = null
    @SerializedName("finished")
    @Expose
    var finished: Boolean? = false


}